### PR TITLE
James Marvel-Coen HW1

### PR DESCRIPTION
**Pull request by jmarvelcoen**
This is my submission of Homework 1.  I've briefly described my process in the README.md file.  I generally followed the step-by-step instructions and used the "Mastering Markdown" guide to help format and write my brief bio, including multiple different elements (bullet points, lists, headers, emphasis, and so forth). Now I am submitting the pull request to complete the assignment.

### CODE REVIEW

**Comment by xt5** [comment]
Evaluation HW 01 for James

Topics                | Excellent            | Satisfactory         |Needs Work      
 ------------------ |-------------------| -------------------|------------------
 Coding Skills    |    x                      |                             |                           
Coding Strategy   |   x    |            |         
Presentation: Graphs   |           |          |         
 Presentation: Tables   |           |             |             
  Achievement, creativity    |  x        |          |            
  Ease of Access       | x      |           |         |     

Remarks:
I agree that following the step-by-step instruction is a good idea. You provided a proper introduction of yourself and demonstrated 4 aspects of the Markdown Syntax. You showed your workflow and I think adding description of how you did the assignment into your HW file is really smart (I should also do that).

**Comment by XianQu2016** [comment]
Coding Skills: Excellent
Coding Strategy: Excellent
Achievement, creativity: Excellent
Ease of Access: Excellent 
Remarks: 
Nice job! You effectively used four style elements: italics, headers, list, and link. Also, the description of steps has proved that you have mastered this workflow. In the commits, it shows that you deleted the line “Repository for first CFSS homework assignment” but kept “hw01”. Even though the header “hw01” could be removed when using Markdown, it still shows that you already mastered those steps and did a good job.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
 # hw01
-Repository for first CFSS homework assignment
```
Comment (XianQu2016) [inline README.md:2]: It shows that you deleted the line “Repository for first CFSS homework assignment” but kept “hw01”. Even though the header “hw01” could be removed when using Markdown, it still shows that you already mastered those steps.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+  
+## James Marvel-Coen
+  
+### Brief Biography
+  
+#### Education and Affiliation
+* B.A., Williams College, 2011-2015
+* University of Chicago, 2015-**present**
+    * I am a second year PhD student in the [Department of Comparative Human Development.](http://humdev.uchicago.edu/)
```
Comment (XianQu2016) [inline README.md:10]: Nice use of italics, headers, list, and link!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
...
+  
+#### Research Interests
+*My research interests include human behavior, cognition, and physiology in the context of evolutionary and developmental processes, and particularly the evolution and development of the human stress response.*
```
Comment (XianQu2016) [inline README.md:13]: Used italics for a whole sentence, creative!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,24 @@
...
+#### How I made this.
+These are the steps I took to make this file:
+
+1. First, I forked the hw01 repo.
+
+2. Then, I created a new project in RStudio > Version Control > Git, and used the URL of the forked hw01 github repository as the repository URL.
+
+3. Then I wrote a brief biography of myself as instructed, and began describing how I did everything.
+
+4. After this, I will commit my changes, push back to github, and then submit a pull request on the original hw01 repo.
```
Comment (XianQu2016) [inline README.md:24]: I like your comment on how you did this step by step! The description of steps has proved that you have mastered this workflow!

**Review by XianQu2016 — APPROVED** [review]
Nice job!!
Coding Skills: Excellent
Coding Strategy: Excellent
Achievement, creativity: Excellent
Ease of Access: Excellent

**Comment by bensoltoff** [comment]
Evaluation HW-01 for James Marvel-Coen
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |       x    |              |            |

Remarks:

* Basic but complete submission. Uses the requested number of Markdown elements and adds a detailed reflection. Nice!
* Make sure in the future to include your reflection as a comment in the pull request, rather than in the readme itself
* There's also no need to commit the HTML file. You can create it to preview on your computer, but when I go to [that page in your repo](https://github.com/jmarvelcoen/hw01-1/blob/master/README.html) all I see is the actual code, not a rendered version. Not very useful. GitHub does all the rendering of Markdown documents (`.md`) for us. Very helpful!

**Pull request closed without merging**